mdata/notifierKafka/cfg.go

```diff
@@ -110,4 +111,16 @@ func ConfigProcess(instance string) {
 			log.Fatal(4, "kafka-cluster: configured partitions not in list of available partitions. missing %v", missing)
 		}
 	}
+
+	// get the "newest" offset for all partitions.
+	// when booting up, we will delay consuming metrics until we have
+	// caught up to these offsets.
+	bootTimeOffsets = make(map[int32]int64)
+	for _, part := range partitions {
+		offset, err := client.GetOffset(topic, part, -1)
+		if err != nil {
+			log.Fatal(4, "kakfa-cluster: failed to get newest offset for %s:%d. %s", topic, part)
```

Comment:
`log.Fatal` basically kills metrictank, right? Or does that get caught by the `recover`, I assume it won't because the log docs say `Fatal is equivalent to Print() followed by a call to os.Exit(1)`.  Do we really want to kill MT if for some reason Kafka is unreachable for a moment? I see that further up in the same file there are quite a few places where `Fatal` is called if there are issues talking to Kafka.